modules/vaos/app/services/vaos/user_service.rb

```diff
@@ -7,7 +7,11 @@ def session
       return cached.token if cached
 
       token = get_session_token
-      SessionStore.new(account_uuid: user.account_uuid, token: token).save
+      session_store = SessionStore.new(account_uuid: user.account_uuid, token: token)
+      session_store.save
+      ttl = get_ttl(token)
```

Comment:
I don't think we lose any clarity if we skip this unnecessary assignment and just call `session_store.expire(get_ttl(token))`.